src/docs/components/trustpilot-stars.hbs

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <title>Trustpilot Stars | Coco &amp; Eve Style Guides</title>
+
+  <link rel="stylesheet" href="/cocoandeve-styleguides/dist/css/main.critical.css">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/dist/css/main.css">
+</head>
+
+<body class="mobile-wrapper">
+  <div class="container mt-4">
+    <h1>Trustpilot Stars</h1>
+    <span class="trustpilot-stars--secondary d-flex mx-auto mx-md-1">
```

Comment:
trustpilot-stars--secondary has no css, also we already have a stars component so stars should go there, all versions